pkgs/servers/mail/cyrus/default.nix

```diff
@@ -0,0 +1,76 @@
+{ stdenv, lib, fetchurl, pkg-config
+, bison, cyrus_sasl, icu, jansson, flex, libbsd, libtool, libuuid, openssl
+, cunit, coreutils, perl
+# CalDAV, CardDAV
+, withHttp ? true, brotli, libical, libxml2, nghttp2, shapelib, zlib
+, withJMAP ? true, libchardet, wslay
+, withXapian ? true, xapian
+, withCalalarmd ? true
+, withMySQL ? false, libmysqlclient
+, withPgSQL ? false, postgresql
+, withSQLite ? true, sqlite
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cyrus";
+  version = "3.4.3";
```

Comment:
Version 3.8.0 is now out:
https://github.com/cyrusimap/cyrus-imapd/releases/tag/cyrus-imapd-3.8.0